src/app/surveys/surveys.component.html

```diff
@@ -39,8 +39,8 @@
         <mat-cell *matCellDef="let row">
           <mat-checkbox (change)="$event ? selection.toggle(row._id) : null"
             [checked]="selection.isSelected(row._id)"
-            [disabled]="(row.teamId && isManagerRoute) || (!isManagerRoute && !row.teamId)"
-            [matTooltip]="getCheckboxTooltip(row)"
```

Comment:
It looks like this utilized that function for to facilitate localizing the tooltip. Is there a reason to switch to setting the tooltip text in the HTML in a way that is not translated?

Reply:
Good catch. I've gone further to move all tooltip handling to the `getActionTooltip` function.